docs/changelog.md

```diff
@@ -7,6 +7,15 @@ The following contains all major, minor, and patch version release notes.
 -   📝 Documentation Update
 -   ⚡ Internal Optimization
 
+## Version 0.1.11
+
+<span class="md-h2-subheader">Release Date: 2025-03-25</span>
+
+-   ✨ Parameterization refactor to support parameter file validation and optional filtering by item or path ([#113](https://github.com/microsoft/fabric-cicd/issues/113))
```

Comment:
Parameterization refactor introducing a new parameter file structure and parameter file validation functionality